tests/vmi_cloudinit_test.go

```diff
@@ -525,7 +518,7 @@ var _ = Describe("[rfe_id:151][crit:high][vendor:cnv-qe@redhat.com][level:compon
 				By("Creating a secret with user and network data")
 				secret := libsecret.New(secretID, libsecret.DataString{"networkdata": testNetworkData})
 
-				_, err := virtClient.CoreV1().Secrets(vmi.Namespace).Create(context.Background(), secret, metav1.CreateOptions{})
+				_, err := virtClient.CoreV1().Secrets(testsuite.GetTestNamespace(vmi)).Create(context.Background(), secret, metav1.CreateOptions{})
```

Comment:
Same here, calling the VMI client create with `testsuite.GetTestNamespace(nil)` gives the same result and it will be consistent with the tests code.

Reply:
please notice that this is not vmi creation, but secret creation. the vmi.NS is not populated at this point.